backpack/extensions/firstorder/sum_grad_squared/conv1d.py

```diff
@@ -1,21 +1,7 @@
-from torch import einsum
+from backpack.core.derivatives.conv1d import Conv1DDerivatives
+from backpack.extensions.firstorder.sum_grad_squared.sum_grad_base import SumGradBase
```

Comment:
Just a minor thing: Can you rename `SumGradBase` into `SGSBase` and `sum_grad_base.py` into `sgs_base.py`